### ISSUE TITLE
top_k_categorical_accuracy seems have problem

### ISSUE THREAD

**Issue by elttaes** [duyongan/sunstreaker#1]
Hi,
My Jax version is 0.3.25, and jnp.logical_or seems not to have reduce.
"match_array = jnp.logical_or.reduce(max_k_preds == y_true, axis=-1)"
https://github.com/duyongan/sunstreaker/blob/main/sunstreaker/metrics.py

**Comment by duyongan** [comment]
Sorry for my late response.
I have fixed this bug.
Sunstreaker haven't  released a stable version  until now.
thanks for your issues.
